<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1135</width>
    <height>658</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="search_box">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>611</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cascadia Code</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:10px;</string>
    </property>
    <property name="text">
     <string>000</string>
    </property>
   </widget>
   <widget class="QComboBox" name="language_switch">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>80</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cascadia Code</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(31, 171, 137);</string>
    </property>
    <item>
     <property name="text">
      <string>C</string>
     </property>
     <property name="icon">
      <iconset theme="QIcon::ThemeIcon::DocumentSend"/>
     </property>
    </item>
    <item>
     <property name="text">
      <string>C++</string>
     </property>
     <property name="icon">
      <iconset theme="QIcon::ThemeIcon::DocumentSend"/>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Rust</string>
     </property>
     <property name="icon">
      <iconset theme="QIcon::ThemeIcon::DocumentSend"/>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Go</string>
     </property>
     <property name="icon">
      <iconset theme="QIcon::ThemeIcon::DocumentSend"/>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Python</string>
     </property>
     <property name="icon">
      <iconset theme="QIcon::ThemeIcon::DocumentSend"/>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Lisp</string>
     </property>
     <property name="icon">
      <iconset theme="QIcon::ThemeIcon::DocumentSend"/>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Assembly</string>
     </property>
     <property name="icon">
      <iconset theme="QIcon::ThemeIcon::DocumentSend"/>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Java</string>
     </property>
     <property name="icon">
      <iconset theme="QIcon::ThemeIcon::DocumentSend"/>
     </property>
    </item>
    <item>
     <property name="text">
      <string>JavaScript</string>
     </property>
     <property name="icon">
      <iconset theme="QIcon::ThemeIcon::DocumentSend"/>
     </property>
    </item>
    <item>
     <property name="text">
      <string>HTML</string>
     </property>
     <property name="icon">
      <iconset theme="QIcon::ThemeIcon::DocumentSend"/>
     </property>
    </item>
    <item>
     <property name="text">
      <string>CSS</string>
     </property>
     <property name="icon">
      <iconset theme="QIcon::ThemeIcon::DocumentSend"/>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Embed</string>
     </property>
     <property name="icon">
      <iconset theme="QIcon::ThemeIcon::DocumentSend"/>
     </property>
    </item>
    <item>
     <property name="text">
      <string>STM32</string>
     </property>
     <property name="icon">
      <iconset theme="QIcon::ThemeIcon::DocumentSend"/>
     </property>
    </item>
    <item>
     <property name="text">
      <string>C51</string>
     </property>
     <property name="icon">
      <iconset theme="QIcon::ThemeIcon::DocumentSend"/>
     </property>
    </item>
    <item>
     <property name="text">
      <string>GIT</string>
     </property>
     <property name="icon">
      <iconset theme="QIcon::ThemeIcon::DocumentSend"/>
     </property>
    </item>
    <item>
     <property name="text">
      <string>QT</string>
     </property>
     <property name="icon">
      <iconset theme="QIcon::ThemeIcon::DocumentSend"/>
     </property>
    </item>
    <item>
     <property name="text">
      <string>MySQL</string>
     </property>
     <property name="icon">
      <iconset theme="QIcon::ThemeIcon::DocumentSend"/>
     </property>
    </item>
    <item>
     <property name="text">
      <string>SQLite</string>
     </property>
     <property name="icon">
      <iconset theme="QIcon::ThemeIcon::DocumentSend"/>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="language_label">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>80</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cascadia Code</family>
      <pointsize>20</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt; color:#5e17eb;&quot;&gt;Language:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QFrame" name="code_frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>511</width>
      <height>481</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QTextEdit" name="code_edit">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>511</width>
       <height>481</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Cascadia Code</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="cursor" stdset="0">
      <cursorShape>IBeamCursor</cursorShape>
     </property>
     <property name="tabStopDistance">
      <double>40.000000000000000</double>
     </property>
    </widget>
    <widget class="QPushButton" name="code_copy">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>10</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(185, 187, 223);
color: rgb(185, 187, 223);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset theme="QIcon::ThemeIcon::EditCopy"/>
     </property>
    </widget>
    <widget class="QPushButton" name="code_save">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>50</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(185, 187, 223);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset theme="QIcon::ThemeIcon::DocumentSave"/>
     </property>
    </widget>
    <widget class="QPushButton" name="code_clear">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>90</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(185, 187, 223);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset theme="QIcon::ThemeIcon::EditClear"/>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="comment_frame">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>140</y>
      <width>501</width>
      <height>481</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QTextEdit" name="comment_edit">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>501</width>
       <height>481</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Cascadia Code</family>
       <pointsize>12</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="cursor" stdset="0">
      <cursorShape>IBeamCursor</cursorShape>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="tabStopDistance">
      <double>40.000000000000000</double>
     </property>
    </widget>
    <widget class="QPushButton" name="comment_copy">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>10</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(185, 187, 223);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset theme="QIcon::ThemeIcon::EditCopy"/>
     </property>
    </widget>
    <widget class="QPushButton" name="comment_save">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>50</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(185, 187, 223);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset theme="QIcon::ThemeIcon::DocumentSave"/>
     </property>
    </widget>
    <widget class="QPushButton" name="comment_clear">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>90</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(185, 187, 223);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset theme="QIcon::ThemeIcon::EditClear"/>
     </property>
    </widget>
    <widget class="QPushButton" name="save_as">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>130</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(185, 187, 223);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset theme="QIcon::ThemeIcon::DocumentSaveAs"/>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="search_button">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>80</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;搜索&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:10px;
background-color: rgb(185, 187, 223);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset theme="QIcon::ThemeIcon::SystemSearch"/>
    </property>
   </widget>
   <widget class="QLabel" name="label_logo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>261</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cascadia Code</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:26pt; font-weight:700; font-style:italic; text-decoration: underline; color:#1fab89;&quot;&gt;Code is Art.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="number_index_browser">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>140</y>
      <width>81</width>
      <height>481</height>
     </rect>
    </property>
   </widget>
   <zorder>search_box</zorder>
   <zorder>language_switch</zorder>
   <zorder>language_label</zorder>
   <zorder>code_frame</zorder>
   <zorder>label_logo</zorder>
   <zorder>comment_frame</zorder>
   <zorder>number_index_browser</zorder>
   <zorder>search_button</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1135</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>search_box</sender>
   <signal>returnPressed()</signal>
   <receiver>search_button</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>436</x>
     <y>116</y>
    </hint>
    <hint type="destinationlabel">
     <x>697</x>
     <y>108</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
